telethon/network/connection/tcpmtproxy.py

```diff
@@ -130,6 +132,21 @@ def address_info(proxy_info):
             raise ValueError("No proxy info specified for MTProxy connection")
         return proxy_info[:2]
 
+    @staticmethod
+    def normilize_secret(secret):
+        if secret[:2] in ["ee","dd"]: # Remove extra bytes
+            secret = secret[2:]
+
+        try:
+            secret_bytes = bytes.fromhex(secret)
+        except:
+            try:
+                secret_bytes = base64.b64decode(secret.encode())
+            except binascii.Error:
+                secret = secret + "=="
+                secret_bytes = base64.b64decode(secret.encode())
+
+        return secret_bytes[:16] # Remove the domain from the secret (until domain support is added)
```

Comment:
Python often adds two spaces before inline comments:

```suggestion
        return secret_bytes[:16]  # Remove the domain from the secret (until domain support is added)
```